src/components/history-page/history-toolbar/keyword-search-input.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { FormControl } from 'react-bootstrap'
+import { useOnInputChange } from '../../../hooks/common/use-on-input-change'
+import { useHistoryToolbarState } from './toolbar-context/use-history-toolbar-state'
+
+/**
+ * A text input that is used to filter history entries for specific keywords.
+ */
+export const KeywordSearchInput: React.FC = () => {
+  const { t } = useTranslation()
+  const [historyToolbarState, setHistoryToolbarState] = useHistoryToolbarState()
+
+  const onChange = useOnInputChange((search) => {
+    setHistoryToolbarState((state) => ({
+      ...state,
+      search
```

Comment:
I'm a big fan of consistency with these types of objects…
```suggestion
      search: search
```

Reply:
Which consistency? We use this format everywhere.

Reply:
Always use `foo: bar` even if it could be omitted due to being the named the same thing.
Like we did with the other components. e.g.:
https://github.com/hedgedoc/react-client/blob/migrate-to-next-js/src/components/history-page/history-toolbar/sort-by-last-visited-button.tsx#L19-L27

Reply:
In your linked case the property name and value are different. I'm with @mrdrogdrog and would keep the ES6 syntax here.

Reply:
Okay, if both of you are against it, then I'll close this thread.